velox/exec/HashTable.cpp

```diff
@@ -189,7 +168,33 @@ class ProbeState {
           }
           return group_;
         }
-        continue;
+      }
+      uint16_t empty = simd::toBitMask(tagsInTable_ == kEmptyGroup) & kFullMask;
+      if (empty > 0) {
+        if (op == Operation::kProbe) {
```

Comment:
I assume there is not logic change in this code block? Thanks!

Reply:
There is no logic change here, the only logic change is line 155-161